src/main/java/org/openlmis/integration/dhis2/web/execution/ManualExecutionController.java

```diff
@@ -66,24 +68,27 @@ public void runExecution() {
     LOGGER.debug("Running manual execution");
     sharedFacilitySynchronizer.refreshSharedFacilities();
     scheduleService.getAllSchedules().forEach(
-        schedule -> processedDataExchangeService.sendData(schedule));
+        schedule -> processedDataExchangeService.sendData(schedule, null, null));
```

Comment:
again null, null. Maybe create another method with only one param

Reply:
I overloaded the sendData method instead of passing nulls in every call.